src/activity-functions-function-notation.xml

```diff
@@ -775,7 +889,107 @@
 
         <p>Solve means to work backwards from a known or given output value and determine all the inputs, if any, that give us the desired output.</p>
 
+        <p>If you live in Oregon in the winter, you might ask, <q>What time will the temperature outside be <m>37</m> degrees fahrenheit?</q> Typically there might be two answers to this question, one time in the morning and another time in the afternoon.  You could answer the question with actual times if you had a time vs. temperature graph or a formula or a table of times and temperatures.</p>
+
         <p>Suppose we want the output of a function to be <m>y =0</m>. Therefore, we must find all the input values of <m>x</m> that gives us that result. There may not be any, but there could be one, several or even infinite possible inputs that gives <m>y = 0</m> as the output.</p>
+        <exercise>
+          <webwork>
+              <title>Solving Graphically</title>
+              <setup>
+                  <var name="$numZeros">
+                      <static>3</static>
+                  </var>
+                  <var name="$zeros">
+                      <static>-4,-1,2</static>
+                  </var>
+                  <var name="$numSolutions1">
+                      <static>2</static>
+                  </var>
+                  <var name="$solutions">
+                      <static>-2.7,2.46</static>
+                  </var>
+                  <var name="$numSolutions2">
+                      <static>1</static>
+                  </var>
+                  <var name="$solutions2">
+                      <static>2.64</static>
+                  </var>
+                  <var name="$maxNumSolutions">
+                      <static>3</static>
+                  </var>
+                   <var name="$minNumSolutions">
+                      <static>1</static>
+                  </var>
+
+
+                  <pg-code>
+                      $numZeros = 3;
+                      $zeros = List(-4,-1,2);
+                      $numSolutions1 = 2;
+                      $solutions = List(-2.7,2.46);
+                      $numSolutions2 = 1;
+                      $solutions2 = List(2.64);
+                      $maxNumSolutions = 3;
+                      $minNumSolutions = 1;
+
+                  </pg-code>
+              </setup>
+
+              <stage>
+                  <statement>
+                      <p>On a graph the output is the height of the graph.  Therefore,<q>Solve <m>f(x) = 0</m></q> on a graph means, <q>Find all the places on the <m>x</m> axis where the graph reaches a height of <m>y = 0</m></q></p>
```

Comment:
`<m>x</m>-axis` (with hyphen) and on the line below, `<m>y</m>-axis` (with m tag)